packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -116,6 +167,62 @@
             }
         }
     }
+
+    &__menu--backtooldui--dtrader {
+        padding: 0 1.5rem 2.5rem;
+        width: 110%;
+    }
+
+    &__menu--back-to-ui {
+        display: flex;
+        width: 90%;
+        justify-content: space-between;
+        border-radius: 1rem;
+    }
+    &__menu--back-to-ui--dark {
+        background-color: var(--general-main-1);
+        display: flex;
+        width: 90%;
+        justify-content: space-between;
+    }
+
+    &__menu--tradinghub {
+        padding: 0 2rem 2.5rem;
+    }
+
+    &__menu--exploretradinghub {
+        display: flex;
+        padding: 1rem;
+        margin: 0 auto;
+        border-radius: 1rem;
+    }
+
+    &__menu--exploretradinghub--dark {
+        display: flex;
+        padding: 1rem;
+        margin: 0 auto;
+        border-radius: 1rem;
+        background-color: var(--general-main-1);
+    }
+
+    &__menu--tradinghub-text {
+        align-items: flex-start;
+    }
+
+    &__menu--back-to-ui-text {
+        align-items: flex-start;
+    }
+    &____menu--back-to-ui-text--dark {
+        color: var(--text-prominent);
+        align-items: flex-start;
+    }
+
+    &__menu-mobile-rightarrow {
+        align-items: flex-end;
+        display: inline-flex;
+        margin-left: 9rem;
+    }
+
```

Comment:
lots of nesting classes can applied in this file to reduce the code lines, please check

Reply:
Will do 👍